build.sh

```diff
@@ -148,6 +152,10 @@ build_deb(){
   # all use the "debian" term, but we want this distribution to be called "one" instead
   sed -i 's@ros-debian-@ros-one-@' $(grep -rl 'ros-debian-' debian/)
   sed -i 's@/opt/ros/debian@/opt/ros/one@g' debian/rules
+  # skip dh_shlibdeps, because some pip modules, speech_recognition for example, contains x86/x86_64/win32/mac binaries
+  sed -i '/dh_shlibdeps / s@$@ || echo "Skip dh_shlibdeps error!!!"@' debian/rules
+  # ignore dh_strip error, from jammy, 'objcopy' added '--compress-debug-sections' and this cause error on 'numpy/core/_multiarray_umath.cpython-310-x86_64-linux-gnu.so has a corrupt string table index - ignoring'
+  echo -e 'override_dh_strip:\n	dh_strip || true\n' |tee -a debian/rules
```

Comment:
The tab string was not correctly inserted in my environment

```shell
printf 'override_dh_strip:\n\t%s\n' 'dh_strip || true' | tee -a debian/rules
```

worked in https://github.com/jsk-ros-pkg/jsk_3rdparty/issues/519 case